src/main/java/me/krumka/onlinebookshop/service/UserServiceImpl.java

```diff
@@ -22,6 +28,11 @@ public UserResponseDto register(UserRegistrationRequestDto userRegistrationReque
             throw new RegistrationException("Cannot register user. Email is already in use.");
         }
         User user = userMapper.toUser(userRegistrationRequestDto);
+        user.setPassword(passwordEncoder.encode(userRegistrationRequestDto.getPassword()));
+        Role.RoleName defaultRoleName = Role.RoleName.ROLE_USER;
+        Role defaultRole = roleRepository.findByName(defaultRoleName)
+                        .orElseThrow(() -> new IllegalStateException("Cannot find default role"));
```

Comment:
add parameter to message Role.RoleName.ROLE_USER
```suggestion
                        .orElseThrow(() -> new EntityNotFoundException("Cannot find default role"));
```